### PR TITLE
Closes #38747: Add italian translations.

### LINKED ISSUE #38747: Missing translations for Italian (it)
Hello,

There are some missing translation for Italian (it). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.it.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.it.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.it.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.it.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by Aerendir**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38747
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by Aerendir** [comment]
Cc @Nyholm

File: src/Symfony/Component/Form/Resources/translations/validators.it.xlf [hunk 1]
```diff
@@ -8,11 +8,131 @@
...
+                <source>Please enter an integer.</source>
+                <target>Per favore, inserisci un numero intero.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Per favore, seleziona una lingua valida.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Per favore, seleziona una lingua valida.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.it.xlf:75]: Make sure this says "locale" and not language.

**Review by Nyholm — COMMENTED** [review]
Thank you. Just one small question

File: src/Symfony/Component/Form/Resources/translations/validators.it.xlf (hunk 1 above)
Comment (Aerendir) [reply src/Symfony/Component/Form/Resources/translations/validators.it.xlf:75]: We don’t have a word for “locale” in Italian. The word “locale” means “room” or “a place where to eat, dance or something similar”.

The word “lingua” means both “locale” and “language”.

We have the verb “localizzare”, that means “find something or someone in a space” and also means “translating”.

But the corresponding noun “locale” is not used with the meaning of a “local language”.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.it.xlf:75]: Okey. Thank you. 

I was just making sure

**Review by Nyholm — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @Aerendir.

**Pull request merged**